src/openMarket/domain/order/Order.js

```diff
@@ -5,19 +5,20 @@ import moment from "moment";
 export default class Order {
 
   /**
-   * @param {IdGenerator} idGenerator
+   * @param {String} id
    * @param {Array.<Line>} lines
+   * @param {date} date
    */
-  constructor({ idGenerator, lines }) {
+  constructor({ id, lines, date=moment().format("DD/MM/YYYY HH:mm:ss") } = {}) {
```

Comment:
I don't understand the part of date :( could you explain in less than 140 characters. Please not brakes :D

Reply:
Yes, it's related with named parameters and default parameters in ES6.
If date key is not provided in constructor, the default value for this key is getting the current date using moment library.
Perhaps you wanna take a look at this http://jimrottinger.github.io/named-arguments-using-es6-destructuring/